src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,64 @@
+package calculator;
+
+import util.Parser;
+
+public class Calculator {
+	private int sum;
+	private Operator activeOperator;
+
+	public Calculator() {
+		this(Operator.PLUS);
+	}
+
+	public Calculator(Operator activeOperator) {
+		this.activeOperator = activeOperator;
+	}
+
+	public int calculate(String formula) {
+		validateFormula(formula);
+		for (String value : formula.split(" ")) {
+			add(value);
+		}
+		return sum;
+	}
+
+	public boolean isNumber(String value) {
+		try {
+			Parser.toInt(value);
+			return true;
+		} catch (Exception e) {
+			return false;
+		}
+	}
+
+	private boolean isCalculatorOperator(String value) {
+		return Operator.containCode(value);
+	}
+
+	public int add(String value) {
+		validateValue(value);
+		if (isNumber(value)) {
+			sum = activeOperator.calculate(sum, Parser.toInt(value));
+		} else if (isCalculatorOperator(value)) {
+			setOperator(value);
```

Comment:
```suggestion
			activeOperator = Operator.valueOfCode(value);
```

메서드의 분리단위가 적어서 바로 호출해도 크게 가독성에 무리 없을것 같습니다.